java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -1113,4 +1115,13 @@ public CompletableFuture<Long> geoadd(
             @NonNull String key, @NonNull Map<String, GeospatialData> membersToGeospatialData) {
         return geoadd(key, membersToGeospatialData, new GeoAddOptions(false));
     }
+
+    @Override
+    public CompletableFuture<Double[][]> geopos(@NonNull String key, @NonNull String[] members) {
```

Comment:
Should the array be NonNull too?
```suggestion
    public CompletableFuture<Double[][]> geopos(@NonNull String key, @NonNull String @NonNull [] members) {
```

Reply:
Right I forgot about that syntax, although for a few of the other commands that take array type arguments the @NonNull tag was also only applied to the element type, do they need to be updated as well?

Reply:
unfortunately, it does not work
I opened a bug for lombok

Reply:
I didn't realize this was another Lombok class. Sorry about that!